packages/frontend/app/styles/components/ilios-header.scss

```diff
@@ -38,7 +38,11 @@
     }
     .user-guide-link {
       grid-area: guide;
-      height: 100%;
+      height: 72%;
```

Comment:
Is this a magic number that looks good, or a calculation based on something else? I'm ok with either, but just wanted to understand where it was coming from. Might need a comment to that effect.

Reply:
I switched this out for a flexbox implementation instead. Thanks for the nudge ;P